src/Badge.svelte

```diff
@@ -1,44 +0,0 @@
-<style>
```

Comment:
If you wanted to keep this hiding it on mobile only would be fine.

Reply:
it doesn't make sense anymore if you're working on it